getting-started/quickstart-guides/php.md

```diff
@@ -42,425 +42,1483 @@ It will prompt a few options. Select defaults by pressing enter.
 composer require imagekit/imagekit
 ```
 
-#### Initialize SDK
+## Quick Examples
+
+#### Create an ImageKit Instance
 
 ```php
 // Import autoloader from vendor
 // If not using PSR-4 is not configured in composer.json file for your project
 require_once __DIR__ . '/vendor/autoload.php';
 
-use ImageKit\ImageKit;
-
+use ImageKit\ImageKit;  
+  
 // For demonstration purposes, the documentation would use https://ik.imagekit.io/demo as urlEndpoint
 $imageKit = new ImageKit(
-    "{publicKey}",
-    "{publicKey}",
-    "{urlEndpoint}" 
+    "publicKey",
+    "privateKey",
+    "urlEndpoint"
 );
-
 ```
+* `publicKey` and `privateKey` parameters are required as these would be used for all ImageKit API, server-side upload, and generating token for client-side file upload. You can get these parameters from the developer section in your ImageKit dashboard - [https://imagekit.io/dashboard#developers](https://imagekit.io/dashboard#developers).
+* `urlEndpoint` is also a required parameter. You can get the value of URL-endpoint from your ImageKit dashboard - [https://imagekit.io/dashboard#url-endpoints](https://imagekit.io/dashboard#url-endpoints).&#x20;
 
-* `urlEndpoint` is the required parameter. You can get the value of URL-endpoint from your ImageKit dashboard - [https://imagekit.io/dashboard#url-endpoints](https://imagekit.io/dashboard#url-endpoints).&#x20;
-* `publicKey` and `privateKey` parameters are also required as these would be used for all ImageKit API, server-side upload, and generating token for client-side file upload. You can get these parameters from the developer section in your ImageKit dashboard - [https://imagekit.io/dashboard#developers](https://imagekit.io/dashboard#developers).
-
-## Generating url for rendering images
-
-#### URL for image with relative path&#x20;
-
+#### URL Generation
 ```php
-$imageURL = $imageKit->url(['path' => '/default-image.jpg']);
-
-echo('Url : ' . $imageURL); 
-// https://ik.imagekit.io/demo/default-image.jpg
+// For URL Generation
+$imageURL = $imageKit->url(
+    [
+        'path' => '/default-image.jpg',
+    ]
+);
+echo $imageURL;
```

Comment:
We should somehow show the value of the URL as well to help the user understand the final output and its relation to the input of this function.
For example, in Node.js SDK we do like below, but the same URL can be put in a comment in the same code snippet in next line after `echo $imageURL`
<img width="877" alt="image" src="https://user-images.githubusercontent.com/45416977/174232775-e97972f7-6609-482d-a851-8d7f0ed94f86.png">